.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
 version: 2.1
 
 orbs:
-  yarn: artsy/yarn@2.1.1
+  artsy-remote-docker: artsy/remote-docker@0.1.0
```

Comment:
Out of curiousity, what is / does `artsy-remote-docker` do?

Reply:
We are attempting to store our docker image so that we can re-use it in workflow steps. The orb builds the image and pushes it to a registry so we can access it later.

Reply:
Ah cool, ty!

Reply:
There's a bit more information here as well: https://github.com/artsy/orbs/tree/master/src/remote-docker